client/src/App/App.js

```diff
@@ -3,7 +3,10 @@ import PropTypes from "prop-types";
 import "./App.css";
 
 const App = ({ children }) => {
-  return <div className="App">{children}</div>;
+  //Make the binding lement semantically correct
+  //Our app is an article, and will have <sections> and <divs>
+  //as children
+  return <article className="app">{children}</article>;
```

Comment:
App is not an article I think. Did you consider using `section`?

Reply:
Sure, create-react-app only bootstraps what is necessarily react dependent to perfection.
IDK if they emphasize on semantic parameters. 
I believe, App itself is the first content /child of the `root` identifier. 

so the `root` becomes `main` and App becomes `article`  - > meaning, an article can have sections and divs. 
A section having an article sounds improper IMO

Reply:
@retrocausal We should definitely maintain HTML semantics. Css style guide makes sense but IMOi think we  should go with `div` Since `article` is that is independent.
`The HTML <article> element represents a self-contained composition in a document, page, application, or site, which is intended to be independently distributable or reusable `

Reply:
thanks @twishasaraiya ! I can change the App element to be a `<div>`. and resubmit the PR. 
does this look good?

`main:root` - > `div:app` -> `div:home`->`section:header`->`section:greensock`->`section:footer` ? 

We do not have the components forming header/greensock/footer yet, but they will need to be `sections` .

Suggestion: 
can we rename the 'home' component? for now since there has been no development on the client side yet, it makes sense to get rid of the redundant `div` inside `App` ? or rename it to header and make it a component instead of a page? 

I am guessing it is better we get the rendering containers organised before we dump data /animations into them

Reply:
hmm.. will look into creating components in a bit, and maybe create the four we require header nav footer and greensock. And will get back to you. can we freeze PRs till we have our containers ready? Will speed it up from my end, and may submit all changes by friday